### ISSUE TITLE
Updating rh-che to the latest 7.0.0-beta-1.0 upstream version on prod

### ISSUE THREAD

**Issue by ibuziuk** [redhat-developer/rh-che#1254]
Related PR https://github.com/redhat-developer/rh-che/pull/1253

**Comment by Katka92** [comment]
@ibuziuk We still have some issues with automated tests running on prod-preview. Today I've tested manually creation and start of workspace (I used vertx and nodejs). Workspace creation and startup worked fine, projects were imported (sometimes with needed refresh of the page) and successfully run. Preview URL shown in console after running project works fine. 
I'm still not able to run Che 7 workspace as shown in screenshot below.
Is there anything else you want me to test manually? 

![Screenshot from 2019-03-12 08-21-06](https://user-images.githubusercontent.com/29278800/54181765-dffe8900-449f-11e9-9eea-5e48ca572bfe.png)
